js/apps/system/_api/foxx/APP/index.js

```diff
@@ -137,7 +137,14 @@ router.get((req, res) => {
 
 router.post(prepareServiceRequestBody, (req, res) => {
   const mount = req.queryParams.mount;
-  FoxxManager.install(req.body.source, mount, Object.assign(
+  let newService;
+  if (req._raw.headers['content-type'] == 'application/javascript' ||
+      req._raw.headers['content-type'] == 'application/zip') {
+    newService = req._raw.requestBody;
```

Comment:
Note to self: this hints at a breaking change in how raw request bodies are handled. Users should never need to access `req._raw` and the request body parsing logic should result in the correct value already being set here.

Reply:
@dothebart FYI Foxx uses `require('internal').rawRequestBody` to get the raw request body internally, which is then parsed as required. I'm guessing this function doesn't work correctly when using vst, which you're trying to work around here by accessing it differently?

Reply:
To be clear: my suggested change would be undoing this diff and instead making sure `require('internal').rawRequestBody` returns `req._raw.requestBody` when using vst.

Reply:
Thanks, found the proper place, removed the clutch.